src/vfShow.ts

```diff
@@ -8,8 +8,23 @@
  * for rendering vexflow. Will eventually go to music21/converter/vexflow
  */
 
-import Vex from 'vexflow';
-
+import {
+    BarlineType,
+    Beam,
+    Flow,
+    Formatter,
+    Fraction,
+    Renderer as VFRenderer,
+    Stave,
+    StaveConnector,
+    StaveNote,
+    StaveTie,
+    SVGContext,
+    TextNote,
+    Tuplet,
+    Voice,
```

Comment:
Another class that also exists in music21  (I am sort of surprised this one wasn't already imported from music21...)